crates/vm/levm/src/vm.rs

```diff
@@ -391,7 +417,21 @@ impl VM {
         }
 
         // Access List Cost
-        // TODO: Implement access list cost.
+        let mut access_lists_cost = U256::zero();
+        for item in self.access_list.clone() {
```

Comment:
Ditto

Reply:
Done in [Change item for (address, keys) in both iterations](https://github.com/lambdaclass/ethrex/pull/1447/commits/142054a8ad143b3d2f3e02851614a82ee5a9accf)